spec/system/user/trusted_user_flags_user_spec.rb

```diff
@@ -16,7 +16,13 @@
     it "does not show the flag button" do
       sign_in create(:user)
 
-      visit user_profile_path(user.username)
+      # [Fix me]: loading the page twice here ensures that the dropdown menu
```

Comment:
do we have a standard FIXME comment format (and is this it?) - if we're using TODO/FIXME tags then using them consistently helps any future effort to find them

Reply:
Great question and suggestion! I just search the codebase and found that it looks like we use `TODO` and `TODO: @<team name>` for general comments regarding, removals, replacements, etc and updated my comment to reflect that.